src/User.ts

```diff
@@ -169,7 +169,7 @@ export class User {
     }
 
     public static fromStorageString(storageString: string): User {
-        Logger.debug("User", "fromStorageString");
+        Logger.debug("User.fromStorageString");
```

Comment:
new Logger("User").create("fromStorageString");

Reply:
I wanted to distinguish static methods from instance methods, so I used the whole property accessor as the name in brackets in these cases.